lib/github/index.js

```diff
@@ -6,8 +7,15 @@ const push = require('./push');
 const { createBranch, deleteBranch } = require('./branch');
 const webhookTimeout = require('../middleware/webhook-timeout');
 
+/**
+ * @param {import('probot').Application} robot - The probot instance
+ */
 module.exports = (robot) => {
+  /**
+   * @param {import('probot').Context} context - The incoming request context.
+   */
   robot.on('*', (context) => {
+    newrelic.setControllerName(`github/events.${context.name}`);
```

Comment:
Would we want `context.event` in here too?

Reply:
Ahh looks like `name` is deprecated in favor of `event` https://github.com/probot/probot/blob/6e45ad7/src/context.ts#L90-L92 I'll update the pr